verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -467,6 +477,12 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
         });
     }
 
+    downloadCases(): void {
+        axios
+            .get('/api/cases/download')
+            .then((response) => fileDownload(response.data, 'cases.csv'));
```

Comment:
this library can't parse the "cases.csv" from the headers in the response? (it has that default name already)

Reply:
No, the fileDownload library is very simple - only takes the data and the filename: https://github.com/kennethjiang/js-file-download

I could parse it from the response headers, but the header it's in is formatted like `content-disposition: attachment; filename="cases.csv"`. We'd have to parse through that string, which doesn't really add benefit IMO.

Reply:
ok no big deal